AdobeStockImage/Model/GetImageList.php

```diff
@@ -8,104 +8,36 @@
 
 namespace Magento\AdobeStockImage\Model;
 
-use Magento\AdobeStockAssetApi\Api\Data\AssetSearchResultsInterface;
-use Magento\AdobeStockAssetApi\Api\Data\AssetSearchResultsInterfaceFactory as SearchResultFactory;
-use Magento\AdobeStockClientApi\Api\ClientInterface;
+use Magento\AdobeStockAssetApi\Api\GetAssetListInterface;
 use Magento\AdobeStockImageApi\Api\GetImageListInterface;
+use Magento\AdobeStockAssetApi\Api\Data\AssetSearchResultsInterface;
 use Magento\Framework\Api\SearchCriteriaInterface;
-use Magento\Framework\Exception\AuthenticationException;
-use Magento\Framework\Exception\LocalizedException;
-use Magento\Framework\Api\Search\Document;
-use Magento\Framework\UrlInterface;
-use Magento\AdobeStockAsset\Model\DocumentToAsset;
-use Psr\Log\LoggerInterface;
 
 /**
  * Class GetImageList
  */
 class GetImageList implements GetImageListInterface
 {
     /**
-     * @var ClientInterface
-     */
-    private $client;
-
-    /**
-     * @var SearchResultFactory
-     */
-    private $searchResultFactory;
-
-    /**
-     * @var DocumentToAsset
+     * @var GetAssetListInterface
      */
-    private $documentToAsset;
-
-    /**
-     * @var UrlInterface
-     */
-    private $url;
-
-    /**
-     * @var LoggerInterface
-     */
-    private $log;
+    private $assetList;
```

Comment:
As this is a command (not a collection) let's name it accordingly to avoid confusion
```suggestion
    private $getAssetList;
```